### ISSUE TITLE
Suggestion: Peaches, for butt size

### ISSUE THREAD

**Issue by Slayerhero90** [naelstrof/KoboldKare#135]
I like butts, and I like fruit puns. It seems like a missing option in the palette of fruit juices, anyways.

Your peach-based pastry could be a Peach Cobbler made with peach juice and dough, perhaps with the side effect of increasing the size of any currently-contained egg, or something else to do with laying.

**Comment by naelstrof** [comment]
Yeah this has been one of the most requested fruit. It's a good idea and I'd like to do it. I should do it, I've only been avoiding it because I've already dialed the kobold's butt to be as big as possible. It also adds more work for people trying to implement their characters to the game-- though butt size dial is a super reasonable one to make.

I'll try to get it in before modding support.
